bitnami/jenkins/values.yaml

```diff
@@ -111,6 +111,23 @@ nodeSelector: {}
 ##
 tolerations: {}
 
+## Jenkins pods' Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+##
+podSecurityContext:
+  fsGroup: 1001
+  runAsUser: 1001
+
+## Jenkins containers' Security Context
```

Comment:
I would clarify that this is only for the container named "jenkins". Using that plural could make it confusing compared to the pod-level security context